src/index.ts

```diff
@@ -27,7 +21,28 @@ function setupOpenMRS() {
   return {
     lifecycle: getAsyncLifecycle(() => import("./root.component"), options),
     activate: (location: Location) =>
-      !location.pathname.startsWith(window.getOpenmrsSpaBase() + "login")
+      !location.pathname.startsWith(window.getOpenmrsSpaBase() + "login"),
+    extensions: [
+      {
+        id: "user-panel-switcher",
```

Comment:
Yeah I'd give it a different ID. Maybe `default-user-panel-switcher` or  `standard-user-panel-switcher`.

I guess what is a good convention is:

- For slot names to describe the location / anticipated content of *where* the slot is
- For components to describe the covered functionality *of* the component itself